qiskit/ignis/verification/randomized_benchmarking/circuits.py

```diff
@@ -177,9 +179,17 @@ def randomized_benchmarking_seq(nseeds=1, length_vector=None,
 
     """
     # Set modules (default is Clifford)
-    if group_gates is None or 'Clifford' or 'clifford':
+    if group_gates is None or group_gates == 'Clifford' or \
```

Comment:
`group_gates` should be an integer (1: clifford, 2: cnotdihedral) you shouldn't need all this cjecking for different spellings, etc

Reply:
I think that it's more intuitive to remember the group name than a number. 
However, I also added the option of a string '0' for the Clifford group, 
and '1' for the CNOT-Dihedral group.

Reply:
At the very least convert it internally to an integer so you only gave to do a multiple or if statement once

Reply:
OK, done.